### PR TITLE
LG-2656, 3556: Stepper reorg + LiveExample width fix

### PR DESCRIPTION
**Pull request by spark33**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [LG-2656](https://jira.mongodb.org/browse/LG-2656)
🎟 _Jira ticket:_ [LG-3556](https://jira.mongodb.org/browse/LG-3556)

### For bug fixes, new features & breaking changes

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] I have run `yarn changeset` and documented my changes

### For new components

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README
- [ ] I have verified the Live Example looks as intended on the design website.

### CODE REVIEW

File: .changeset/twelve-poets-kick.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+'@leafygreen-ui/stepper': patch
```
Comment (shaneeza) [inline .changeset/twelve-poets-kick.md:2]: I think we're marking file restructuring as `major`. @brookeyalof do you know?

Comment (bruugey) [reply .changeset/twelve-poets-kick.md:2]: I think it's just a patch

File: packages/stepper/src/index.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
-import Step from './Step';
-import Stepper from './Stepper';
-import type { StepperProps } from './types';
+import Step from './Step/Step';
+import Stepper from './Stepper/Stepper';
+import type { StepperProps } from './Stepper/Stepper.types';
```
Comment (shaneeza) [inline packages/stepper/src/index.ts:3]: Some of your sub components have index files and some don't. To keep it consistent you could add index files to `Step` and `Stepper`, add your exports there, and then import them from `./Stepper` and `./Step`

**Comment by shaneeza** [comment]
did you check out how stepper looks on .design?

**Comment by shaneeza** [comment]
@spark33 in case the comment above was missed, did you check that Stepper looks good on .design as well?

File: packages/stepper/src/EllipsesStep/EllipsesStep.styles.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import { spacing } from '@leafygreen-ui/tokens';
+
+import { stepIconClassName } from '../constants';
+
+import { EllipsesStepState, EllipsesStepStates } from './EllipsesStep.types';
+
+// TODO: would be good to put these styles inside a baseStyles, but it is currently impossible because the <Tooltip> content is an iframe.
```
Comment (shaneeza) [inline packages/stepper/src/EllipsesStep/EllipsesStep.styles.ts:10]: iframe?

Comment (spark33) [reply packages/stepper/src/EllipsesStep/EllipsesStep.styles.ts]: Hahahah this is Sean on his first project being confused. Someone probably explained the Portal in Tooltip as "like an iframe" or something and I just soaked it up

**Pull request merged**